app/controllers/responses_controller.rb

```diff
@@ -19,8 +19,10 @@ def index
   end
 
   def new
+    # set the locale based upon the default_langauge of the survey
     I18n.locale = @survey.settings["default_language"].to_sym || I18n.default_locale
```

Comment:
Is this only for the public side of the interface? This means that if the user completes the survey in a certain language, by selecting this in the dropdown, the results page won't necessarily be shown in that language. Is this correct and intended?

Reply:
This is a good point... let me double check! Thanks for pointing this out 😄